CHARTER.md

```diff
@@ -1,25 +1,16 @@
-# Open Container Initiative Charter v1.1
+# Open Container Initiative Charter v1.2
 
 | Comment          | Effective Date   |
 | ---------------- |:----------------:|
 | Initial release  | 13 November 2015 |
-| Update           | 6 May 2020       |
+| Update           | 18 May 2020       |
 
 ##  1. Mission of the Open Container Initiative (“OCI”).
 
-The Open Container Initiative provides an open source technical community within which industry participants may easily contribute to building vendor-neutral, portable and open specifications and runtime that deliver on the promise of containers as a source of application portability backed by a certification program.
+The Open Container Initiative provides an open source technical community within which industry participants may easily contribute to building vendor-neutral, portable and open specifications, reference implementations, and tools that deliver on the promise of containers as a source of application portability backed by a certification program.
```

Comment:
This is a good interim change, but we really should have a proper rewrite of some of the more substantial parts of the charter. I'll send a draft PR sometime this week with the sorts of changes I'd propose us having.